### PR TITLE
Format

### PR DESCRIPTION
**Pull request by atefBB**
formating !

### CODE REVIEW

File: bin/ungit [hunk 1]
```diff
@@ -58,11 +58,11 @@ const launch = () => {
   );
 
   child.on('exit', (res) => {
-    console.log('Stopped keeping ungit alive');
+    console.log('Stopped! keeping ungit alive');
   });
 
   const startupListener = (data) => {
-    if (data.toString().indexOf('## Ungit started ##') >= 0) {
```
Comment (jung-kim) [inline bin/ungit]: Ungit's clicktest capture this console output to detect the service start and this is the reason why all tests are failing.

Comment (atefBB) [reply bin/ungit]: @jung-kim how to change this behavior?

Comment (jung-kim) [reply bin/ungit]: Before that, why do you want to change this behavior?

Comment (atefBB) [reply bin/ungit]: the phrase `## Ungit started ##` with `##` seems ugly for me !

Comment (jung-kim) [reply bin/ungit]: it seems that what is being suggested here is about the appearance of string, I'm not sure that is a strong enough of a reason to change test behaviors.

Comment (atefBB) [reply bin/ungit]: As you want! I think it's not that difficult to change test behavior! 
Have a nice day.

**Pull request closed without merging**